api/src/pcapi/routes/backoffice_v3/templates/offerer/get/details/managed_venues.html

```diff
@@ -0,0 +1,31 @@
+<table class="table table-hover my-4">
+    <thead>
+        <tr>
+            <th scope="col"></th>
+            <th scope="col">ID</th>
+            <th scope="col">Nom</th>
+            <th scope="col">SIRET</th>
+            <th scope="col">Type de lieu</th>
+        </tr>
+    </thead>
+
+    <tbody>
+        {% for venue in offerer.managedVenues %}
+            <tr>
+                <th scope="row"></th>
+                <td class="fw-bolder">
+                    <a href="{{ url_for("backoffice_v3_web.venue.get", venue_id=venue.id) }}">
+                        {{ venue.id }}
+                    </a>
+                </td>
+                <td class="text-break">
+                    <a href="{{ build_pc_pro_venue_link(venue) }}">
+                        {{ venue.name }}
```

Comment:
(détail) Je suggère d'afficher de préférence le `publicName` s'il existe. Quelque chose comme : 

```suggestion
                        {{ venue.publicName or venue.name }}
```

Reply:
Corrigé :)